### ISSUE TITLE
[Suggestion] Add played opening to "Recent Games" 

### ISSUE THREAD

**Issue by tom-anders** [lichess-org/mobile#947]
Short anecdote to give some motivation for this feature: A few days ago, I played a few games of Rapid. As always, in one of the games I got completely caught off-guard by the French Defense. The next day, I remembered that I wanted to analyze the French defense game, so I went to the new app and tried to find it in my recent games. In the old app, this was very easy, as it displayed the played opening (and even a thumbnail of the final position) for each played game. The new app currently does not have this information, making it hard to find a specific game sometimes.

So I'd propose to add the opening info to recent games. I can understand that the board thumbnail would probably take away too much space, but I think we could at least add the opening while still keeping the entries rather compact. As always, I'd be happy to have a go at implementing this, if you think it's a feature we should have :)

**Comment by julien4215** [comment]
Just a suggestion if you want to add the board thumbnail. I think it would be a good idea to allow the user to switch between to views : the actual one with the list and another one with board thumbnails.

To allow the user to switch between the two views, you could add this [icon](https://api.flutter.dev/flutter/cupertino/CupertinoIcons/rectangle_grid_1x2-constant.html) next to the filter one for the list view and this [one](https://api.flutter.dev/flutter/cupertino/CupertinoIcons/square_grid_2x2-constant.html) for the grid view.

**Comment by ijm8710** [comment]
I agree with this but it would also be nice to have more advanced options to filter by opening type. Even if all the options from desktop weren't built for the app, to at least have a way to select them from a mobile redirect that then filters back within the app if that was possible.

**Comment by veloce** [comment]
I agree that the opening is an important information, so feel free to have a go to improve the game list.

**Comment by HaonRekcef** [comment]
In my opinion, the opening information is not significant enough to clutter the games list. The name of the opening as well as the final position can already be accessed by long-pressing the games in the list. 
However, I really like the idea of a switch between different views.

**Comment by tom-anders** [comment]
Yeah for me it would also be fine to display the opening only in the grid/detail view, but not in the (default) list view. Let's see what I can come up with. I'd imagine the new grid view to have a similar layout as the old app's recent games view

**Comment by ijm8710** [comment]
How id see it is defaulted  off for the 10 games on the homepage. But defaulted on once you click more to see full archive game list

**Comment by tom-anders** [comment]
Agreed, on the homepage we should probably keep the list as it is currently.

Quick mockup of what the grid screen could look like:

![Screenshot_1725050806](https://github.com/user-attachments/assets/16291093-e2b4-4e84-9e2a-6e6a50159e77)

One thing that's missing is probably rating (diff)

**Comment by ijm8710** [comment]
And computer analysis available

**Comment by EmmetSchuler** [comment]
Accuracy of analyzed games would be a welcome addition as well

**Comment by veloce** [comment]
I'd see a slightly smaller board. Also in order to not clutter the screen I'd change things a little. How about not having the 10+0 at the top but only the variant icon and the date on the same line in a small font?

If we switch to such a screen the header in the context menu would be redundant and should be revised too I guess.
